docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -793,247 +917,449 @@ know the exact order or values of the array itself.
+
+Filter Using Tag Timestamp Values
++++++++++++++++++++++++++++++++++
+
+A tag timestamp can be thought of as a specialized "property" of a tag that happens to be a date / time range
+(interval). You can filter nodes based on tag timestamp values using any comparison operator supported by interval
+(:ref:`type-ival` types). The time / interval extended operator ( ``@=`` ) is used most often, but equal to
+( ``=`` ) can also be used to **exactly** match the values in the interval.
+
+See :ref:`filter-interval` for additional detail on the use of the ``@=`` operator.
+
+.. NOTE::
+  
+  It is not currently possible to filter generically based on "tags that have timestamps" vs. "tags that do not
+  have timestamps".
+
+**Syntax:**
+
+*<query>* **+** | **-** **#** *<tag>* **@=** *<time>* | **(** *<min_time>* **,** *<max_time>* **)**
+
+*Filter the current result set to only include nodes that were associated with anonymous VPN infrastructure between December 1, 2023 and January 1, 2024:*
+
+.. storm-pre:: [ (inet:fqdn=wazn.airservers.org +#cno.infra.anon.vpn=(2023/05/24 23:16:51.423, 2023/12/05 23:12:40.626)) (inet:ipv6=2607:9000:0:85:68a3:75b4:13ab:770a +#cno.infra.anon.vpn=(2023/08/15 00:12:15,2023/12/05 23:12:54) ) ] +#cno.infra.anon.vpn @= ( 2023/12/01, 2024/01/01 )
 ::
-    
-    <query> +#cno.infra.dns.sinkhole@=(2017/01/01, 2018/01/01)
+  
+  <query> +#cno.infra.anon.vpn @= ( 2023/12/01, 2024/01/01 )
+
+*Filter the current working set to only include nodes that were owned / controlled by Threat Cluster 15 as of October 30, 2021:*
 
+.. storm-pre:: [ inet:fqdn=ronthecat.com +#cno.threat.t15.own=(2020/09/12, 2022/09/12) ] +#cno.threat.t15.own @= 2021/10/30
+::
+  
+  <query> +#cno.threat.t15.own @= 2021/10/30
+
+.. _filter-tag-prop:
+
+Filter Using Tag Properties
++++++++++++++++++++++++++++
 
-- Filter results to exclude nodes associated with threat cluster 17 after January 1, 2019:
+:ref:`tag-properties` can be used to provide additional context to tags. Storm supports filtering nodes
+whose tags have a specific tag property (regardless of the value of the property).
 
+.. NOTE::
+  
+  In many cases, information previously recorded using a tag property is better suited to the use of
+  an :ref:`gloss-extended-prop`.
 
-.. storm-pre:: inet:fqdn -#cno.threat.t17.own@=(2019/01/01, now)
+
+**Syntax:**
+
+*<query>* **+** | **-** **#** *<tag>* **:** *<tagprop>*
+
+
+*Filter the current working set to only include nodes with a ":risk" property reported by Symantec:*
+
+.. storm-pre:: [ (inet:fqdn=woot.com +#rep.symantec:risk=87) (inet:ipv4=8.8.8.8 +#rep.domaintools:risk=42 ) ] +#rep.symantec:risk
 ::
-    
-    <query> -#cno.threat.t17.own@=(2019/01/01, now)
+  
+  <query> +#rep.symantec:risk
+  
+.. NOTE::
+  
+  You must specify a tag associated with the tag property. It is not possible to filter nodes based on a particular
+  tag property being present on **any** tag (e.g., Storm queries such as ``+#:risk`` will generate ``BadSyntax`` errors).
```

Comment:
You actually can do this now, the syntax for this filter now allows wildcards in tags so `+#*:risk` works (still not allowed if a tagprop value is specified though)

Reply:
(also I know there's a separate story for docs on the wildcard/interfaces stuff, just pointing this one out so it doesn't need to be removed later)